test/k8sT/Updates.go

```diff
@@ -121,6 +133,37 @@ func InstallAndValidateCiliumUpgrades(kubectl *helpers.Kubectl, ciliumFilename,
 	apps := []string{helpers.App1, helpers.App2, helpers.App3}
 	app1Service := "app1-service"
 
+	ciliumCiliumState := func(helmPath, helmVersion string) {
+		_ = kubectl.ExecMiddle("helm delete cilium --namespace=" + helpers.CiliumNamespace)
+		_ = kubectl.ExecMiddle(fmt.Sprintf("kubectl delete configmap --namespace=%s cilium-config", helpers.CiliumNamespace))
```

Comment:
Maybe move all `kubectl delete` instances into a separate function which could be later re-used in `AfterAll()` to remove all Cilium-related artifacts?

Reply:
@brb it's already the case, this is called in line 103

Reply:
I meant in `AfterAll()` of other test suites. I.e. to move to the `test/helpers/`.